src/modules/getFood.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove it & solve the linter manually, It will make the code cleaner